packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -138,6 +140,7 @@ export class DeviceSession implements Disposable {
     }
 
     await Promise.all([
+      this.metro.ready(),
```

Comment:
why is it here? isn't `this.devtools.appReady()` promise dependent on metro process?

Reply:
it is not, I added a note about that in the PR description